Your request could not processed. Please press the back button on your browser and try again.<br/>If the problem persists, please contact technical support<p/>Information below is for technical support:<p/>URL: http://localhost:4435/pta_proxy/Admin/Register.aspx<p/>Querystring:<p/><p>-------------------------<p/>Form:<p/><br/>__EVENTTARGET: <br/><br/>__EVENTARGUMENT: <br/><br/>__VIEWSTATE: a4bGR6wzpMfcFBT2ThJeeKMOfrLkighky+Lt6sD4U9rc9GWJBVGRjfozK7XXvm3cP26sBSClxqmEwRWSLhF66G8nkkg/WDIzSCrVpgRnuvrUI/fqmutU+VO2/p5BuNdYM2e0eio0DOhgfMcLjb0Hdekp90aQVWS1wyRj+Sl6UqZY5e7XQRBnc74gGfXXvRnPSR3y4ZFPKC/Z2gUEbml3s9TOWLRp5O2kfWxlsuW2n+KYE19Ma3vByQfrGSM77U03bQvEHm/bdIzwcmLtkBYG4jJnWL7GhPG8K6GgH7aF4r0A6gBrpac3iR7wouQgObWOhq19t6fWdoSEx/I5ve6LefF70rmHEtQ64s8yfe7W67s4Qg1toXuCXelP5q2NvykuROtbQm83F+LYmQ65BvcStlK5T987NiU1FLoBMPbV9iOY2WaYbcLJ8SC7jZPQhwysbRurjF6hAbO6kfH7vDiFw6/f+PgOMWYPV/yP5Sb0WMLTx88vh/LAQNZlM8/btLNA4JlwWPE+QerqIX2sSc+iRTU4+JcPQrbgOeJ1oS/7rfji25zBQku5Ww/PVRYqo4fV5Cv2bt8H63AaKP9R+IXyLui9Kn718sNfGONAHG9UcYQLpcO1olmdrodslVQ1sxSRfvgh408vj+JB8rxhGFmzEO9vMlSaP47AEEnJ7htnGWsspKMuB7edOiHoIzoz7j5Ymcf6wX2Qtlc8S8YUasZEUB8ohqx5fxnh9YBjajz/ByBvbYc/hk/OMhNt4v1drySgkxkgqwIEdPEEnr9L7bfPRnZY86Xyc3byGG9jJFvVQhPdszptu+G2ucSSj9tIkUD9Vk1PiGmCnZESSCfcoObyY4KSG2AusSuCoDv4xz0q+k3AqdjICRPegXFDth/OyoElmzW7GvYYQqn4CTxDff4/1/CmIr43mVFePu0YQEVbqZMh3CeO20dSJBmq0eDxSc1E5NLlzyizVI3TttuBasZkVu1JJf0+GEAzU4oSSiboJmWQ/afPw4OCGK66si7Ju56msqyuzf//tnChgYkV70Mep4dnyaI/rmBfcz4UlmU696I=<br/><br/>__VIEWSTATEENCRYPTED: <br/><br/>__EVENTVALIDATION: DKckc3DPy2k3qtBXsoSbpwcqYDNjKKDYQRnv9durJEMlyrmWC1g4tED0y7Gpq+LMRb5Qd3doA7ov4GnjUMkjTGl0cdHLMBplBH9YzWawj9e/sj/FbLX2cHGx8QjESOHeAyeJKTgrWd1HGqU2eX4LyjreCC8JsgacF+hWN6AWZTYuQO2looY6tvDyc0E+FX1tflNFPpMbIH3dSk+AJhN5dcGoCf/rOcrWW5dVUONWWAsh3dAW/yGjTRrd+WcgZGnX<br/><br/>ddlBBoard: idlatest.com<br/><br/>txtBBRegPwd: <br/><br/>txtRegXml: <br/><br/>btnRegister: Register<br/><br/>txtXML: <br/><p>-------------------------<p/>ErrorInfo:<p/><p/> Error Message:<br/>Exception of type 'System.Web.HttpUnhandledException' was thrown.<p/>Stacktrace:<br/><PRE>   at System.Web.UI.Page.HandleError(Exception e)
   at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   at System.Web.UI.Page.ProcessRequest(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)
   at System.Web.UI.Page.ProcessRequest()
   at System.Web.UI.Page.ProcessRequestWithNoAssert(HttpContext context)
   at System.Web.UI.Page.ProcessRequest(HttpContext context)
   at ASP.admin_register_aspx.ProcessRequest(HttpContext context) in d:\W2KS\Microsoft.NET\Framework\v2.0.50727\Temporary ASP.NET Files\pta_proxy\ab85bea8\a60dc983\App_Web_qanexg2g.0.cs:line 0
   at System.Web.HttpApplication.CallHandlerExecutionStep.System.Web.HttpApplication.IExecutionStep.Execute()
   at System.Web.HttpApplication.ExecuteStep(IExecutionStep step, Boolean& completedSynchronously)</PRE><br/><p/> InnerException.Error Message:<br/>The underlying connection was closed: An unexpected error occurred on a receive.<p/>Stacktrace:<br/><PRE>   at System.Web.Services.Protocols.WebClientProtocol.GetWebResponse(WebRequest request)
   at System.Web.Services.Protocols.HttpWebClientProtocol.GetWebResponse(WebRequest request)
   at Microsoft.Web.Services3.WebServicesClientProtocol.GetResponse(WebRequest request, IAsyncResult result)
   at Microsoft.Web.Services3.WebServicesClientProtocol.GetWebResponse(WebRequest request)
   at System.Web.Services.Protocols.SoapHttpClientProtocol.Invoke(String methodName, Object[] parameters)
   at ContextWS.initialize() in W:\BB-PTAuth\src\pta_proxy_VS2008_WSE\pta_proxy\AppCode\ContextWS.cs:line 444
   at Idla.PtaProxy.WSWrapper.initializeInternal() in W:\BB-PTAuth\src\pta_proxy_VS2008_WSE\pta_proxy\AppCode\Idla.PtaProxy\WSWrapper.cs:line 81
   at Idla.PtaProxy.WSWrapper.initialize() in W:\BB-PTAuth\src\pta_proxy_VS2008_WSE\pta_proxy\AppCode\Idla.PtaProxy\WSWrapper.cs:line 63
   at Idla.PtaProxy.WSCall.DoCall(WSCall ws_call) in W:\BB-PTAuth\src\pta_proxy_VS2008_WSE\pta_proxy\AppCode\Idla.PtaProxy\WSCall.cs:line 66
   at Idla.PtaProxy.WSCall.RegisterTool(String bbUrl, String toolRegistrationPassword) in W:\BB-PTAuth\src\pta_proxy_VS2008_WSE\pta_proxy\AppCode\Idla.PtaProxy\WSCall.cs:line 77
   at Idla.PtaProxy.Admin.Register.btnRegister_Click(Object sender, EventArgs e) in W:\BB-PTAuth\src\pta_proxy_VS2008_WSE\pta_proxy\Admin\Register.aspx.cs:line 38
   at System.Web.UI.WebControls.Button.OnClick(EventArgs e)
   at System.Web.UI.WebControls.Button.RaisePostBackEvent(String eventArgument)
   at System.Web.UI.WebControls.Button.System.Web.UI.IPostBackEventHandler.RaisePostBackEvent(String eventArgument)
   at System.Web.UI.Page.RaisePostBackEvent(IPostBackEventHandler sourceControl, String eventArgument)
   at System.Web.UI.Page.RaisePostBackEvent(NameValueCollection postData)
   at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)</PRE><br/><p/> InnerException.InnerException.Error Message:<br/>Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host.<p/>Stacktrace:<br/><PRE>   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.Net.FixedSizeReader.ReadPacket(Byte[] buffer, Int32 offset, Int32 count)
   at System.Net.Security._SslStream.StartFrameHeader(Byte[] buffer, Int32 offset, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security._SslStream.StartReading(Byte[] buffer, Int32 offset, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.Security._SslStream.ProcessRead(Byte[] buffer, Int32 offset, Int32 count, AsyncProtocolRequest asyncRequest)
   at System.Net.TlsStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.Net.PooledStream.Read(Byte[] buffer, Int32 offset, Int32 size)
   at System.Net.Connection.SyncRead(HttpWebRequest request, Boolean userRetrievedStream, Boolean probeRead)</PRE><br/><p/> InnerException.InnerException.InnerException.Error Message:<br/>An existing connection was forcibly closed by the remote host<p/>Stacktrace:<br/><PRE>   at System.Net.Sockets.Socket.Receive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags)
   at System.Net.Sockets.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 size)</PRE><br/><p>-------------------------<p/>Request body:<p/><br/>__EVENTTARGET=&__EVENTARGUMENT=&__VIEWSTATE=a4bGR6wzpMfcFBT2ThJeeKMOfrLkighky%2BLt6sD4U9rc9GWJBVGRjfozK7XXvm3cP26sBSClxqmEwRWSLhF66G8nkkg%2FWDIzSCrVpgRnuvrUI%2FfqmutU%2BVO2%2Fp5BuNdYM2e0eio0DOhgfMcLjb0Hdekp90aQVWS1wyRj%2BSl6UqZY5e7XQRBnc74gGfXXvRnPSR3y4ZFPKC%2FZ2gUEbml3s9TOWLRp5O2kfWxlsuW2n%2BKYE19Ma3vByQfrGSM77U03bQvEHm%2FbdIzwcmLtkBYG4jJnWL7GhPG8K6GgH7aF4r0A6gBrpac3iR7wouQgObWOhq19t6fWdoSEx%2FI5ve6LefF70rmHEtQ64s8yfe7W67s4Qg1toXuCXelP5q2NvykuROtbQm83F%2BLYmQ65BvcStlK5T987NiU1FLoBMPbV9iOY2WaYbcLJ8SC7jZPQhwysbRurjF6hAbO6kfH7vDiFw6%2Ff%2BPgOMWYPV%2FyP5Sb0WMLTx88vh%2FLAQNZlM8%2FbtLNA4JlwWPE%2BQerqIX2sSc%2BiRTU4%2BJcPQrbgOeJ1oS%2F7rfji25zBQku5Ww%2FPVRYqo4fV5Cv2bt8H63AaKP9R%2BIXyLui9Kn718sNfGONAHG9UcYQLpcO1olmdrodslVQ1sxSRfvgh408vj%2BJB8rxhGFmzEO9vMlSaP47AEEnJ7htnGWsspKMuB7edOiHoIzoz7j5Ymcf6wX2Qtlc8S8YUasZEUB8ohqx5fxnh9YBjajz%2FByBvbYc%2Fhk%2FOMhNt4v1drySgkxkgqwIEdPEEnr9L7bfPRnZY86Xyc3byGG9jJFvVQhPdszptu%2BG2ucSSj9tIkUD9Vk1PiGmCnZESSCfcoObyY4KSG2AusSuCoDv4xz0q%2Bk3AqdjICRPegXFDth%2FOyoElmzW7GvYYQqn4CTxDff4%2F1%2FCmIr43mVFePu0YQEVbqZMh3CeO20dSJBmq0eDxSc1E5NLlzyizVI3TttuBasZkVu1JJf0%2BGEAzU4oSSiboJmWQ%2FafPw4OCGK66si7Ju56msqyuzf%2F%2FtnChgYkV70Mep4dnyaI%2FrmBfcz4UlmU696I%3D&__VIEWSTATEENCRYPTED=&__EVENTVALIDATION=DKckc3DPy2k3qtBXsoSbpwcqYDNjKKDYQRnv9durJEMlyrmWC1g4tED0y7Gpq%2BLMRb5Qd3doA7ov4GnjUMkjTGl0cdHLMBplBH9YzWawj9e%2Fsj%2FFbLX2cHGx8QjESOHeAyeJKTgrWd1HGqU2eX4LyjreCC8JsgacF%2BhWN6AWZTYuQO2looY6tvDyc0E%2BFX1tflNFPpMbIH3dSk%2BAJhN5dcGoCf%2FrOcrWW5dVUONWWAsh3dAW%2FyGjTRrd%2BWcgZGnX&ddlBBoard=idlatest.com&txtBBRegPwd=&txtRegXml=&btnRegister=Register&txtXML=<br/>